_overviews/tutorials/scala-for-java-programmers.md

```diff
@@ -160,38 +160,49 @@ package, so can be accessed from anywhere in a program.
 
 > **Note:** The following assumes you are using Scala on the command line
 
+If we save the above program in a file called
+`HelloWorld.scala`, we can run it by issuing the following
+command (the greater-than sign `>` represents the shell prompt
+and should not be typed):
+
+```shell
+> scala run HelloWorld.scala
+```
+
+The program will be automatically compiled (with compiled classes somewhere in the newly created `.scala-build` directory)
+and executed, producing a similar output:
```

Comment:
```suggestion
and executed, producing an output similar to:
```
("a similar output" in English refers to another output that was *previously* mentioned; there is no such thing here)